WebDriverAgentTests/UnitTests/Doubles/XCUIElementDouble.h

```diff
@@ -27,6 +27,7 @@
 @property (nonatomic, readwrite, getter=isWDEnabled) BOOL wdEnabled;
 @property (nonatomic, readwrite, getter=isWDVisible) BOOL wdVisible;
 @property (nonatomic, readwrite, getter=isWDAccessible) BOOL wdAccessible;
+@property (nonatomic, readwrite, getter=isWDFocused) BOOL wdFocused;
```

Comment:
@KazuCocoa We probably would need to put this under ifdef?

Reply:
Yes, it looks safer to add it

Reply:
added